src/components/PrivateRoute/component.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react'
+import { Layout } from 'antd'
 import { Route, Redirect } from 'react-router-dom'
 import PropTypes from 'prop-types'
+import Header from '../../containers/Header'
 
 const PrivateRoute = ({ component: Component, sessionID, ...rest }) => (
   <Route
     {...rest}
     render={props => (
-      sessionID ? <Component {...props} /> : <Redirect to="/login" />
+      sessionID ? (
+        <>
+          <Layout>
```

Comment:
i am not sure this is correct place for layout. it would be better if you will use it inside top level component or router